<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallControlWidget</class>
 <widget class="QFrame" name="CallControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>30</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="lblTimeIcon">
       <property name="text">
        <string notr="true">TI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTime">
       <property name="text">
        <string notr="true">00:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>41</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="tlbDialerVisible">
     <property name="text">
      <string notr="true">D</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tlbCameraEnable">
     <property name="text">
      <string notr="true">C</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tlbMicrophoneEnable">
     <property name="text">
      <string notr="true">M</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="sdrSpeakerVolume">
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tlbReject">
     <property name="text">
      <string notr="true">Reject</string>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>41</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="tlbCameraPreview">
     <property name="text">
      <string notr="true">P</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
